test/integration/exercise11_response_test.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require_relative '../test_helper'
+
+require 'pp'
+
+require_relative '../examples/exercise11_object_nest'
+
+module Examples
+  class Exercise11Controller < ExampleController
+    def initialize
+      self.params = { }
+    end
+
+    def request
+      OpenStruct.new(query_parameters: params, url: 'http://test.api/exercise11')
+    end
+  end
+end
+
+module Integration
+  class Exercise11ResponseTest < MiniTest::Test
+    context 'Response to Exercise11' do
+      setup do
+        Flappi.configure do |conf|
+          conf.version_plan = nil
+        end
+
+        @controller = Examples::Exercise11Controller.new
+      end
+
+      should 'respond with instrument code when include true' do
+        @controller.params = { include: true }
+        response = @controller.show
+
+        assert_equal({ 'a' => 5, 'instrument' => { 'code' => 'AAA' } }, response)
+      end
+
+      should 'respond with nil instrument when include false' do
+        @controller.params = { include: false }
+        response = @controller.show
+
+        assert_equal({ 'a' => 15 }, response)
```

Comment:
But here I think it should be `{ 'a' => 15, 'instrument' => nil}`, shouldn't it? @richdrich

Reply:
No, it omits it. For a javascript / json object, this is the same thing I think.

Reply:
You could use a dummy field if you really want nil

Reply:
@felixibel Yeah, this is fine to me too.  `instrument: nil` and `instrument` not being present (`= undefined`) are the same as far as consumption goes.